features/score/src/test/java/org/opennms/alec/features/score/impl/ScoringStrategyTest.java

```diff
@@ -76,13 +78,16 @@ public void canComputeScores() {
         report = scorer.score(Sets.newHashSet(emtpySituation), Sets.newHashSet(emtpySituation));
         assertThat(report.getScore(), IsCloseTo.closeTo(0.0d, delta));
 
-        // Comparing a situation with a single alarm to an empty situation should generate a score greater than 0
-        Situation situation = ImmutableSituation.newBuilderNow()
-                .setId("test")
-                .addAlarm(ImmutableAlarm.newBuilder().setId("test").build())
-                .build();
-        report = scorer.score(Sets.newHashSet(situation), Sets.newHashSet(emtpySituation));
-        assertThat(report.getScore(), Matchers.greaterThan(0.0d));
+        // ARI and AMI won't pass this test unless we add missing alarms as singletons
+        if(!scorer.getName().equals("ari") && !scorer.getName().equals("ami")){
+            // Comparing a situation with a single alarm to an empty situation should generate a score greater than 0
+            Situation situation = ImmutableSituation.newBuilderNow()
+                    .setId("test")
+                    .addAlarm(ImmutableAlarm.newBuilder().setId("test").build())
+                    .build();
+            report = scorer.score(Sets.newHashSet(situation), Sets.newHashSet(emtpySituation));
+            assertThat(report.getScore(), Matchers.greaterThan(0.0d));
```

Comment:
I'd consider adding the `if` block around the assertion and testing for 0 in this case.

Reply:
yup 0 is the expected score here so this sounds good for now. we may want to discuss how cluster engines and scores deal with singletons in the future.